src/spotted/data/user.py

```diff
@@ -95,6 +106,40 @@ def sban(self) -> bool:
             return True
         return False
 
+    def mute(self, bot: Bot, days: int = 0):
+        """_summary_
+
+        Args:
+            bot (Bot): the telegram bot
+            days (int, optional): The number of days the user should be muted for.
+                                    Defaults to 0.
+        """
+        bot.restrict_chat_member(
+            chat_id=Config.post_get("channel_id"),
+            user_id=self.user_id,
+            can_send_messages=False,
+            can_send_media_messages=False,
+            can_send_other_messages=False,
+            can_add_web_page_previews=False,
+        )
+        current_date_time = datetime.now()
+        expiration_date_time = (current_date_time + timedelta(days=days)).strftime("%Y-%m-%d %H:%M:%S")
+        DbManager.insert_into(
+            table_name="muted_users",
+            columns=("user_id", "expiration_date"),
+            values=(self.user_id, expiration_date_time),
+        )
```

Comment:
Is this feature never utilized? Because I can't see the `muted_users` table anywhere

Reply:
This is WIP, the idea is to add a /mute command